platforms/common/include/px4_platform_common/defines.h

```diff
@@ -95,7 +95,11 @@ __BEGIN_DECLS
 extern long PX4_TICKS_PER_SEC;
 __END_DECLS
 
+#if defined(CONFIG_ARCH_BOARD_MODALAI_VOXL2)
```

Comment:
We should avoid board specifics like this slipping into common headers, can we get this set another way either explicitly configured (kconfig) or derived from the CWD or something?

Reply:
I'll see if I can find a better way.

Reply:
I added a Kconfig for Posix platforms to configure it.

Reply:
But now a bunch of checks are failing. Maybe due to the include of px4_platform_common/px4_config.h in defines.h? But seems like a real error. A file is using a variable _K but there is a macro with that same name.

Reply:
Okay, looks like I fixed the failing test cases by being more accurate in my header file inclusion.